src/mumble/AudioOutput.cpp

```diff
@@ -588,29 +588,42 @@ bool AudioOutput::mix(void *outbuff, unsigned int frameCount) {
 						aop->pfVolume[s] = -1.0;
 				}
 
+				if (!aop->piOffset) {
+					aop->piOffset = new unsigned int[nchan];
```

Comment:
If this variable is turned into a smart-pointer, then this code here should use `std::make_unique` instead of `new`